doc/examples/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -1,198 +1,99 @@
-:moveit1:
-
-..
-   Once updated for MoveIt 2, remove all lines above title (including this comment and :moveit1: tag)
-
 Perception Pipeline Tutorial
-============================
+==================================
+
+Introduction
+------------
+MoveIt serves sensor plugins for utilizing 3D sensor data in planning pipeline. When it's ran the demo of this tutorial, it should be seen the following rviz screen.
 
-MoveIt allows for seamless integration of 3D sensors using `Octomap <http://octomap.github.io/>`_.
-Once properly configured, you should see something like this in rviz:
+.. raw:: html
 
-.. image:: perception_configuration_demo.png
-   :width: 700px
+        <video width="700px" controls="true" autoplay="true" loop="true">
+            <source src="../../../_static/videos/perception_pipeline_demo.webm" type="video/webm">
+            MoveIt Perception Pipeline Demo
+        </video>
 
 Getting Started
 ---------------
-If you haven't already done so, make sure you've completed the steps in :doc:`Getting Started </doc/tutorials/getting_started/getting_started>`.
-
-Configuration
--------------
-
-In this section, we will walk through configuring the 3D sensors on your robot with MoveIt. The primary component in MoveIt that deals with 3D perception is the Occupancy Map Updater. The updater uses a plugin architecture to process different types of input. The currently available plugins in MoveIt are:
-
-* The PointCloud Occupancy Map Updater: which can take as input point clouds (``sensor_msgs/PointCloud2``)
-
-* The Depth Image Occupancy Map Updater: which can take as input Depth Images (``sensor_msgs/Image``)
-
-To use the Occupancy Map Updater, it is only necessary to set the appropriate parameters on the ROS parameter server and to call ``startWorldGeometryMonitor`` from your ``PlanningSceneMonitor``.  This latter step is performed automatically when using Move Group functionality like in this tutorial's example, so in that case it is only necessary to set the parameters for the octomap and octomap updater.
-
-YAML Configuration file (Point Cloud)
-+++++++++++++++++++++++++++++++++++++
-
-We will have to generate a YAML configuration file for configuring the 3D sensors. Please see :panda_codedir:`this example file<config/sensors_kinect_pointcloud.yaml>` for processing point clouds.
-
-Save this file in the config folder in the robot's moveit_config package with name "sensors_kinect_pointcloud.yaml": ::
-
- sensors:
-   - sensor_plugin: occupancy_map_monitor/PointCloudOctomapUpdater
-     point_cloud_topic: /camera/depth_registered/points
-     max_range: 5.0
-     point_subsample: 1
-     padding_offset: 0.1
-     padding_scale: 1.0
-     max_update_rate: 1.0
-     filtered_cloud_topic: filtered_cloud
-
-**The general parameters are:**
```

Comment:
Can we keep the param descriptions too? You could merge them as a comment with the `sensors_3d.yaml: ::` description

Reply:
I have added to https://github.com/CihatAltiparmak/moveit2_tutorials/blob/bc1504d3d6c13e17c19bf506fe70358bbc83a100/doc/examples/perception_pipeline/perception_pipeline_tutorial.rst?plain=1#L90-L108 . I will also add for depth map plugin.